<?xml version="1.1"?>

<launch>
  <rosparam file="$(find wj_716n_lidar)/params/ros2_wj_716N_cfg.yaml" command="load"/>
  <node name="wj_716N_lidar_01" pkg="wj_716n_lidar" type="wj_716n_lidar" respawn="false" output="screen">
  </node>
</launch>

